services/editing/src/org/domokit/editing/KeyboardImpl.java

```diff
@@ -39,6 +39,11 @@ public void setClient(KeyboardClient client, KeyboardConfiguration configuration
     }
 
     @Override
+    public void clearClient() {
+        mViewState.close();
```

Comment:
Yes - KeyboardViewState.close() drops the KeyboardClient held by the KeyboardViewState.  (I'm not sure why it was named "close" - maybe because it closes the KeyboardClient mojo stub?)